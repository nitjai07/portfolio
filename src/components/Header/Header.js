import React from 'react';

const Header = () => (
  <header>
    <p>NJ Portfolio</p>
  </header>
);

export default Header;
