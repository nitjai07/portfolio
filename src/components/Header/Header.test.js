import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';

test('Header component renders properly', () => {
  const component = shallow(<Header />);
  expect(component).toMatchSnapshot();
});
