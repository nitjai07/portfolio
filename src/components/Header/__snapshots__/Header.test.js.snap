// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders properly 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <header>
    <p>
        NJ Portfolio
    </p>
</header>,
  "nodes": Array [
    <header>
      <p>
            NJ Portfolio
      </p>
</header>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_context": Object {},
    "_element": <Header />,
    "_forcedUpdate": false,
    "_instance": null,
    "_newState": null,
    "_rendered": <header>
      <p>
            NJ Portfolio
      </p>
</header>,
    "_rendering": false,
    "_updater": Updater {
      "_callbacks": Array [],
      "_renderer": [Circular],
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Header />,
}
`;
